app/uk/gov/hmrc/individualsdetailsapi/connectors/IfConnector.scala

```diff
@@ -47,6 +47,8 @@ class IfConnector @Inject()(servicesConfig: ServicesConfig, http: HttpClient)(
     servicesConfig.getString(
       "microservice.services.integration-framework.environment")
 
+  private val cidClientId = servicesConfig.getString("cid-client-id")
```

Comment:
Do we need to do a get with option and default this to `SOMETESTVALUE` for local dev? Just so its not reliant on the config?